### ISSUE TITLE
TWW Prepatch Errors

### ISSUE THREAD

**Issue by yajinni** [Legacy-of-Sylvanaar/wow-instant-messenger#86]
436x WIM/Sources/WindowHandler.lua:1924: attempt to call upvalue 'GetMouseFocus' (a nil value)
[string "@WIM/Sources/WindowHandler.lua"]:1924: in function `fun'
[string "@WIM/Sources/WindowHandler.lua"]:437: in function <WIM/Sources/WindowHandler.lua:429>
[string "@WIM/Sources/WindowHandler.lua"]:753: in function <WIM/Sources/WindowHandler.lua:753>

Locals:
self = WIM3_msgFrame1ExitButton {
 widgetName = "close"
 parentWindow = WIM3_msgFrame1 {
 }
 curTextureIndex = 1
 forceShift = false
}
SelectedSkin = <table> {
 tab_strip = <table> {
 }
 title = "WIM Classic"
 emoticons = <table> {
 }
 website = "http://www.wimaddon.com"
 author = "Pazza <Bronzebeard>"
 message_window = <table> {
 }
 version = "1.0.0"
}
(*temporary) = nil
(*temporary) = "attempt to call upvalue 'GetMouseFocus' (a nil value)"
WIM = <table> {
 addonTocName = "WIM"
 tblInUse = <table> {
 }
 modules = <table> {
 }
 windows = <table> {
 }
 debug = false
 Menu = WIM3Menu {
 }
 lastState = "other"
 db = <table> {
 }
 lists = <table> {
 }
 useProtocol2 = true
 ctxMenu = <table> {
 }
 constants = <table> {
 }
 L = <table> {
 }
 O = <table> {
 }
 S = <table> {
 }
 ChatUserList = WIM3_ChatUserList {
 }
 isModernApi = true
 env = <table> {
 }
 tblAvailable = <table> {
 }
 curState = "other"
 nextColor = <table> {
 }
 isInitialized = true
 history = <table> {
 }
 filters = <table> {
 }
 armoryLinks = <table> {
 }
 db_defaults = <table> {
 }
 WindowParent = WIM_UIParent {
 }
 options = <table> {
 }
 Notifications = <table> {
 }
 beta = false
 chatFilters = <table> {
 }
 libs = <table> {
 }
 stats = <table> {
 }
 EditBoxInFocus = WIM3_msgFrame1MsgBox {
 }
 windowsByAge = <table> {
 }
 version = "3.11.1-1-g18c69d4"
}
GetMouseFocus = nil

**Comment by Jaidy88** [comment]
same:


864x WIM/Sources/WindowHandler.lua:1924: attempt to call upvalue 'GetMouseFocus' (a nil value)
[string "@WIM/Sources/WindowHandler.lua"]:1924: in function `fun'
[string "@WIM/Sources/WindowHandler.lua"]:437: in function <WIM/Sources/WindowHandler.lua:429>
[string "@WIM/Sources/WindowHandler.lua"]:753: in function <WIM/Sources/WindowHandler.lua:753>

Locals:
self = WIM3_msgFrame1ExitButton {
 widgetName = "close"
 parentWindow = WIM3_msgFrame1 {
 }
 curTextureIndex = 1
 forceShift = false
}
SelectedSkin = <table> {
 tab_strip = <table> {
 }
 title = "WIM ElvUI (Solid)"
 website = "https://www.curseforge.com/members/fuba82/projects"
 author = "fuba"
 message_window = <table> {
 }
 version = "1.0.0"
}
(*temporary) = nil
(*temporary) = "attempt to call upvalue 'GetMouseFocus' (a nil value)"
WIM = <table> {
 addonTocName = "WIM"
 tblInUse = <table> {
 }
 modules = <table> {
 }
 windows = <table> {
 }
 debug = false
 Menu = WIM3Menu {
 }
 lastState = "other"
 db = <table> {
 }
 lists = <table> {
 }
 useProtocol2 = true
 ctxMenu = <table> {
 }
 constants = <table> {
 }
 L = <table> {
 }
 O = <table> {
 }
 S = <table> {
 }
 ChatUserList = WIM3_ChatUserList {
 }
 isModernApi = true
 env = <table> {
 }
 tblAvailable = <table> {
 }
 curState = "other"
 nextColor = <table> {
 }
 MinimapIcon = WIM3MinimapButton {
 }
 isInitialized = true
 history = <table> {
 }
 filters = <table> {
 }
 armoryLinks = <table> {
 }
 db_defaults = <table> {
 }
 WindowParent = WIM_UIParent {
 }
 options = <table> {
 }
 Notifications = <table> {
 }
 beta = false
 chatFilters = <table> {
 }
 libs = <table> {
 }
 stats = <table> {
 }
 windowsByAge = <table> {
 }
 version = "3.11.2"
}
GetMouseFocus = nil

**Comment by Floppy012** [comment]
Same error in a more readable way:
```
Message: Interface/AddOns/WIM/Sources/WindowHandler.lua:1924: attempt to call upvalue 'GetMouseFocus' (a nil value)
Time: Wed Jul 24 17:19:33 2024
Count: 7810
Stack: Interface/AddOns/WIM/Sources/WindowHandler.lua:1924: attempt to call upvalue 'GetMouseFocus' (a nil value)
[string "@Interface/AddOns/WIM/Sources/WindowHandler.lua"]:1924: in function `fun'
[string "@Interface/AddOns/WIM/Sources/WindowHandler.lua"]:437: in function <Interface/AddOns/WIM/Sources/WindowHandler.lua:429>
[string "@Interface/AddOns/WIM/Sources/WindowHandler.lua"]:753: in function <Interface/AddOns/WIM/Sources/WindowHandler.lua:753>

Locals: self = WIM3_msgFrame1ExitButton {
 widgetName = "close"
 parentWindow = WIM3_msgFrame1 {
 }
 curTextureIndex = 1
 forceShift = false
}
SelectedSkin = <table> {
 tab_strip = <table> {
 }
 title = "WIM Classic"
 emoticons = <table> {
 }
 website = "http://www.wimaddon.com"
 author = "Pazza <Bronzebeard>"
 message_window = <table> {
 }
 version = "1.0.0"
}
(*temporary) = nil
(*temporary) = "attempt to call upvalue 'GetMouseFocus' (a nil value)"
WIM = <table> {
 addonTocName = "WIM"
 tblInUse = <table> {
 }
 modules = <table> {
 }
 windows = <table> {
 }
 debug = false
 Menu = WIM3Menu {
 }
 lastState = "resting"
 MSG_CONTEXT_MENU_EDITBOX = WIM3_msgFrame1MsgBox {
 }
 db = <table> {
 }
 lists = <table> {
 }
 useProtocol2 = true
 ctxMenu = <table> {
 }
 constants = <table> {
 }
 L = <table> {
 }
 O = <table> {
 }
 S = <table> {
 }
 ChatUserList = WIM3_ChatUserList {
 }
 isModernApi = true
 env = <table> {
 }
 tblAvailable = <table> {
 }
 curState = "resting"
 nextColor = <table> {
 }
 MinimapIcon = WIM3MinimapButton {
 }
 isInitialized = true
 history = <table> {
 }
 filters = <table> {
 }
 armoryLinks = <table> {
 }
 db_defaults = <table> {
 }
 WindowParent = WIM_UIParent {
 }
 options = <table> {
 }
 Notifications = <table> {
 }
 beta = false
 chatFilters = <table> {
 }
 libs = <table> {
 }
 stats = <table> {
 }
 windowsByAge = <table> {
 }
 version = "3.11.2"
}
GetMouseFocus = nil
```

**Comment by karpana** [comment]
I'm getting two different errors via BugSack.

```lua
317x WIM/Modules/MinimapIcon.lua:348: attempt to call upvalue 'GetMouseFocus' (a nil value)
[string "@WIM/Modules/MinimapIcon.lua"]:348: in function <WIM/Modules/MinimapIcon.lua:347>

Locals:
self = Frame {
}
(*temporary) = nil
(*temporary) = "attempt to call upvalue 'GetMouseFocus' (a nil value)"
icon = WIM3MinimapButton {
 flash = WIM3MinimapButtonFlash {
 }
 icon = Texture {
 }
 text = FontString {
 }
 backGround = Texture {
 }
}
GetMouseFocus = nil
```

and this one as well
```lua
1531x WIM/Sources/WindowHandler.lua:359: attempt to call upvalue 'GetMouseFocus' (a nil value)
[string "@WIM/Sources/WindowHandler.lua"]:359: in function <WIM/Sources/WindowHandler.lua:357>

Locals:
self = WIM_WindowHelperFrame {
 flash = WIM_WindowHelperFrameFlash {
 }
}
(*temporary) = nil
(*temporary) = "attempt to call upvalue 'GetMouseFocus' (a nil value)"
GetMouseFocus = nil
WIM = <table> {
 addonTocName = "WIM"
 tblInUse = <table> {
 }
 modules = <table> {
 }
 windows = <table> {
 }
 debug = false
 Menu = WIM3Menu {
 }
 lastState = "other"
 db = <table> {
 }
 lists = <table> {
 }
 useProtocol2 = true
 ctxMenu = <table> {
 }
 constants = <table> {
 }
 L = <table> {
 }
 O = <table> {
 }
 S = <table> {
 }
 ChatUserList = WIM3_ChatUserList {
 }
 isModernApi = true
 env = <table> {
 }
 tblAvailable = <table> {
 }
 curState = "other"
 nextColor = <table> {
 }
 MinimapIcon = WIM3MinimapButton {
 }
 isInitialized = true
 history = <table> {
 }
 filters = <table> {
 }
 armoryLinks = <table> {
 }
 db_defaults = <table> {
 }
 WindowParent = WIM_UIParent {
 }
 options = <table> {
 }
 Notifications = <table> {
 }
 beta = false
 chatFilters = <table> {
 }
 libs = <table> {
 }
 stats = <table> {
 }
 windowsByAge = <table> {
 }
 version = "3.11.2"
}
resizeFrame = WIM_WindowResizeFrame {
 widgetName = "resize"
}
```

**Comment by MelchiorGaspar** [comment]
ok great so other gamer(s) have already report it.

**Comment by Pazza** [comment]
Fixed in 3.12.0

**Comment by MelchiorGaspar** [comment]
ty so much for the quick fixes Pazza
